### ISSUE TITLE
Greeting action failing

### ISSUE THREAD

**Issue by rmlarose** [unitaryfund/mitiq#577]
From #574, copying here so we don't forget. https://github.com/unitaryfund/mitiq/runs/2018759271

cc @crazy4pi314 

```
with:
    repo-token: ***
    pr-message: Hello @Aaron-Robertson, thank you for submitting a PR! We will respond as soon as possible, and if you have any questions in the meantime, you can ask us on the [Unitary Fund Discord](https://discord.unitary.fund).
    issue-message: Hello @Aaron-Robertson, thank you for your interest in Mitiq!
  If this is a bug report, please provide screenshots and/or **minimum viable code to reproduce your issue**,  so we can do our best to help get it fixed. If you have any questions in the meantime, you can also ask us on the [Unitary Fund Discord](https://discord.unitary.fund).
/usr/bin/docker run --name e48eb7546109d549348d3ac495d82c8ac2_b1134e --label 5588e4 --workdir /github/workspace --rm -e INPUT_REPO-TOKEN -e INPUT_PR-MESSAGE -e INPUT_ISSUE-MESSAGE -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_RETENTION_DAYS -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e GITHUB_ACTION_REPOSITORY -e GITHUB_ACTION_REF -e GITHUB_PATH -e GITHUB_ENV -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/_temp/_runner_file_commands":"/github/file_commands" -v "/home/runner/work/mitiq/mitiq":"/github/workspace" 5588e4:8eb7546109d549348d3ac495d82c8ac2
[@octokit/rest] `const Octokit = require("@octokit/rest")` is deprecated. Use `const { Octokit } = require("@octokit/rest")` instead
[@octokit/rest] `const Octokit = require("@octokit/rest")` is deprecated. Use `const { Octokit } = require("@octokit/rest")` instead
Checking if its the users first contribution
Checking...
Checking...
Checking...
Checking...
Adding message: Hello @Aaron-Robertson, thank you for submitting a PR! We will respond as soon as possible, and if you have any questions in the meantime, you can ask us on the [Unitary Fund Discord](https://discord.unitary.fund). to pull request 574
Error: Resource not accessible by integration
```

**Comment by nathanshammah** [comment]
Since we decided not to focus on CI issues in this milestone, but it is quite critical to have CI not failing, my proposal is to remove the `greeting.yml` document right now from the code base, and address how to solve this in a future milestone in a more complex way. If there are no objections @crazy4pi314 @rmlarose @andreamari, I'll proceed.

**Comment by rmlarose** [comment]
Could be a very quick fix, I would wait for @crazy4pi314 before removing. 

Note: If further commits are pushed to the PR, the greeting message doesn't trigger and thus passes.

**Comment by crazy4pi314** [comment]
Yeah it would only fail the very first time a user ever makes a PR and no other time. I will look into this on stream today and let you know if we should yank it.

**Comment by rmlarose** [comment]
This was fixed in #617.

**Comment by rmlarose** [comment]
See https://github.com/unitaryfund/mitiq/pull/618#pullrequestreview-626771612 for an example of the greeting action working.
